### PR TITLE
Ignore some PHP specific code on 32 bit, fixing CI

### PR DESCRIPTION
**Pull request by bwoebi**
Fixes i686 windows CI

### CODE REVIEW

File: sidecar/src/interface.rs [hunk 1]
```diff
@@ -480,7 +480,10 @@ impl EnqueuedTelemetryData {
                     }
                     async fn parse(path: &PathBuf) -> Result<Vec<data::Dependency>> {
                         let mut json = tokio::fs::read(&path).await?;
+                        #[cfg(not(target_arch = "x86"))]
                         let parsed: ComposerPackages = simd_json::from_slice(json.as_mut_slice())?;
+                        #[cfg(target_arch = "x86")]
```
Comment (bantonsson) [inline sidecar/src/interface.rs:485]: Which project is interested in x86?

**Review by bantonsson — APPROVED** [review]

File: sidecar/src/interface.rs (hunk 1 above)
Comment (bwoebi) [reply sidecar/src/interface.rs:485]: I don't know, the sidecar isn't. Just making it pass here.

**Comment by r1viollet** [comment]
Should we not prevent the full sidecar build on non x86 ?

**Comment by bwoebi** [comment]
@r1viollet I don't care - the sidecar generally should still work, this a PHP specific routine where the x86 problem occurs.

**Comment by r1viollet** [comment]
> @r1viollet I don't care - the sidecar generally should still work, this a PHP specific routine where the x86 problem occurs. And if yes, please provide a PR, not sure how to do that.

I see your point that this is not the focus. I am worried that we are creating new branches that should be tested in a different way.
Fixing CI is more important right now.

**Pull request merged**